.taskcluster.yml

```diff
@@ -140,18 +144,18 @@ tasks:
                     path: '/tmp/image.tar'
                     type: 'file'
               scopes:
-                $if: 'PUSH_DOCKER_IMAGE == "0"'
+                $if: 'push_docker_image == "0"'
                 then: []
                 else:
                   - 'secrets:get:project/releng/scriptworker-scripts/deploy'
               metadata:
                 $let:
                   description:
-                    $if: 'PUSH_DOCKER_IMAGE == "0"'
+                    $if: 'push_docker_image == "0"'
                     then: '${project_name}-py${python_version}: build docker image [on ${DOCKER_TAG}]'
                     else: '${project_name}-py${python_version}: build and push docker image [on ${DOCKER_TAG}]'
                 in:
-                  name: '${description}'
+                  name: '${i}.2. ${description}'
```

Comment:
Why is this 2?

Reply:
this will nicely order the tasks. number 2 is the dependent task of the Run tox. and since the taskcluster UI orders by name we can take the advantage of this to have things nicely ordered.